api/openapi.yaml

```diff
@@ -1449,17 +1449,12 @@ paths:
           schema:
             type: integer
             default: 0
-        - in: query
```

Comment:
Right now, we are not explicitly putting a `query` parameter on our endpoints because they are injected as it is on the queries, making our endpoints unsafe. Later we need to improve this.

Reply:
IMO we should put it here and fix the implementation

Reply:
Agreed. We can open a PR to solve that.

Reply:
Issue created: https://github.com/kubeshop/tracetest/issues/2388